IviDriverCore/1.0/Spec/IviDriverCore.md

```diff
@@ -0,0 +1,770 @@
+
+The format of the version shall follow the rules for FileVersion defined in [File Versioning](#file-versioning) followed by an optional string. If the string is present, a space shall separate the version from the string. The string contains additional driver specific version information. Multi-byte characters are not allowed in the string that this property returns. String characters shall be in the range of `\x20` - `\x7E`.
+
+Examples of allowed version strings are shown below:
+
+    4.00.1
+
+    02.0001.12345.1 This version adds XYZ capability to the component
+
+### Driver Vendor
+
+Returns the name of the vendor that supplies the IVI Core driver.
+
+### Error Query
+
+The Error Query member queries the instrument and returns instrument specific error information.
+
+For instruments that implement an error queue, Error Query extracts a single error from the queue.
+
+For instruments that have status registers but no error queue, the IVI driver returns an error consistent with instrument design.
+
+The operation of Error Query is not affected by the value of Query Instrument Status Enabled.
+
+### Instrument Manufacturer
+
+Returns the name of the manufacturer of the instrument. The IVI driver returns the value it queries from the instrument or a string indicating that it cannot query the instrument identity. For instance, "Cannot query from instrument".
+
+### Instrument Model
+
+Returns the model number or name of the instrument. The IVI driver returns the value it queries from the instrument or a string indicating that it cannot query the instrument identity. For instance, "Cannot query from instrument".
+
+### Query Instrument Status Enabled
+
+IVI drivers shall implement Query Instrument Status Enabled if possible. At *initialization* Query Instrument Status Enabled shall be false, unless *initialization* options have been used to enable it.
+
+If the instrument can be queried for its status and Query Instrument Status Enabled is True, then the driver checks the instrument status at the end of every call by the user to a method that accesses the instrument and reports an error if the instrument has detected an error. If False, the driver does not query the instrument status at the end of each user operation.
+
+If the instrument status cannot be meaningfully queried after an operation, then this property has no effect on driver operation.
+
+> **Observation:**
+> > Querying the instrument status is very useful for debugging. After validating the program, the user can set this property to False to disable status checking and maximize performance.
+
+### Reset
+
+Reset performs the following actions:
+
+- Places the instrument in a known state. For instance, an IEEE 488.2 instrument driver may send the command string `*RST` to the instrument.
+
+- Configures instrument options on which the IVI driver depends. A driver might enable or disable headers or enable binary mode for waveform transfers.
+
```

Comment:
Reset should also set all configuration (properties) to its default and clear the hardware memory, right? We should call it out.